### DISCUSSION TITLE
Remove padding while in editor

### DISCUSSION THREAD

**Discussion by timskovjacobsen** [kovidgoyal/kitty#5733] [Q&A] [answered]
I have added a padding to my kitty conf as per #584, which I like for standard shell use. However, that padding stays around when I enter a terminal editor, like vim.

Does anyone know if it is possible to reset the padding to 0 while in an editor?

**Comment by kovidgoyal** [comment, marked as answer]
Use remote control

kitty @ set-spacing

**Reply by timskovjacobsen** [reply]
> Use remote control kitty @ set-spacing

Thanks. I'll try to adjust the margin via `kitty @ set-spacing margin=<SOME_VALUE>` upon opening vim and then find a way to reset the margin when going back to the terminal.

**Reply by bwkam** [reply]
My problem with this approach is that the editor doesn't run at all when in a terminal multiplexer, like tmux. Is there a way to workaround this?

**Reply by kovidgoyal** [reply]
Preferably dont use a terminal multiplexer, but if you must then setup remote control via a socket to bypass the multiplexer breaking remote control.

**Reply by bwkam** [reply]
Thank you.
